playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -100,8 +100,8 @@
   hosts: oo_masters_to_config:oo_nodes_to_config
   vars:
     target_version: "{{ '1.2' if deployment_type == 'origin' else '3.1.1.900' }}"
-    openshift_docker_hosted_registry_insecure: True
-    openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
+    docker_hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(True) }}"
```

Comment:
If we tried to do something like this here we'd get into a template loop BUT we require that both of these variables be set in playbooks prior to including a role that includes `openshift_docker`.

```
openshift_docker_hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(True) }}"
```

I think we could remove all `openshift_docker_hosted_registry_insecure` settings from playbooks and let `openshift_docker_facts` default the variable to `True` OR we could default to `True` wherever the value is accessed if the intent was to not persist this fact when the user didn't provide it.

We will continue to require passing `openshift_docker_hosted_registry_insecure` since it's driven by the first master's `portal_net` but this isn't a variable that is exposed to users so renaming here makes sense to me.